fe/lib/components/SalaryDetails/SalaryDetails.stories.tsx

```diff
@@ -52,16 +44,8 @@ export default {
       control: { type: 'text' },
     },
   },
-  component: SalaryDetailsComponent,
-  title: 'Job Posting/Salary details',
```

Comment:
This one looks to be a definite change

Reply:
Ah yeah that and the `SeekApiResponse` story will now have different titles. I changed the title here so the nesting would line up with everything else, but I'll change it back to how it originally was to fix this. The changes the `SeekApiResponse shouldn't matter I don't think since theres no mock component for it.

Reply:
Nope, I accidentally removed it, will add back in 👍